packages/search/README.md

```diff
@@ -1,88 +1,161 @@
+    <sp-help-text slot="negative-help-text">Invalid search term.</sp-help-text>
+</sp-search>
+```
+
+#### Clear button
+
+A clear button is a button that clears the search term. It is displayed when the search field is not empty.
+
+```html demo
+<sp-search value="Flights to Hawaii"></sp-search>
+```
+
+### States
+
+#### Disabled
+
+A search field in a disabled state shows that a search option exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that it may become available later.
+
+```html demo
+<sp-search disabled></sp-search>
+```
+
+### Events
+
+The `submit` event fires when the `<sp-search>` is submitted. This is a non-`composed` event in line with what you would expect a [`<form />`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) to fire. If you choose to prevent the default behavior of this event, the default action for the underlying `<form />` element will also be prevented, allowing you to handle the search action in JavaScript.
+
+### Accessibility
+
+#### Include a label
+
+Every text field should have a label. A field without a label is ambiguous and not accessible.
+
+The `aria-label` for the search field is set to `Search` by default. The text set in the `label` property is used as the `aria-label` for the search field.
+
+#### Include help text
+
+The description in the help text is flexible and encompasses a range of guidance. Sometimes this guidance is about what to input, and sometimes it's about how to input. This includes information such as:
+
+- An overall description of the input field
+- Hints for what kind of information needs to be input
+- Specific formatting examples or requirements
+
+Learn more about [using help text](https://spectrum.adobe.com/page/text-field/#Use-help-text-to-show-hints,-formatting,-and-requirements).
+
+#### Include negative help text
+
+Write error messaging in a human-centered way by guiding a user and showing them a solution — don't simply state what's wrong and then leave them guessing as to how to resolve it. Ambiguous error messages can be frustrating and even shame-inducing for users. Also, keep in mind that something that a system may deem an error may not actually be perceived as an error to a user.
+
+Learn more about [writing error messages](https://spectrum.adobe.com/page/text-field/#Write-error-text-that-shows-a-solution).
 
-The `submit` event fires when the `<sp-search>` is submitted. This is a non-`composed` event inline with what you would expect a [`<form />`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event){:target="\_blank"} to fire. If you choose to prevent the default of this event, the default action for the underlying `<form />` element will also be prevented, which will allow you to handle the search action in javascript.
+#### Do not use a placeholder as a replacement for a label or help-text
```

Comment:
Love this as a heading call-out. Very important point and well detailed. Thanks!